readme.md

```diff
@@ -125,6 +125,7 @@
 - [NodeOS](https://github.com/NodeOS/NodeOS) - The first operating system powered by npm.
 - [YodaOS](https://github.com/yodaos-project/yodaos) - AI operating system.
 - [Brain.js](https://github.com/BrainJS/brain.js) - Machine-learning framework.
+- [Pipcook](https://github.com/alibaba/pipcook) - Front-end algorithm framework to create a ML pipeline based on tfjs-node.
```

Comment:
tfjs-node is deprecated: https://github.com/tensorflow/tfjs-node

`ML` => machine learning